packages/openshift/context.py

```diff
@@ -493,17 +498,24 @@ def tracking(action_handler=None):
     performed, this method will be called with the Action object.
     If not specified, all Actions will aggregate into a internally
     managed Result object which can be accessed with get_result.
+    :param limit: If specified, it allows to specify a limit on the
+    number of actions stored by a given tracking context. If not specified
+    will store unlimited number of oc interactions.
     :return: The tracker contextmanager. If action_handler is not
     specified, call get_result to receive a Result object with all
     tracked Action objects.
     """
     c = Context()
+    if limit:
```

Comment:
`0` in python is falsey, so consider what your developer would expect if they set limit=0 and what they would get instead.

Reply:
The idea was to not let them keep it 0 but i see your point. Should i raise a warning for 0 or by default keep it a standard value ?